README.md

```diff
@@ -20,15 +20,16 @@ This is the official Sentry SDK for [Sentry].
 
 To use Sentry in your project, add it as a dependency in your `mix.exs` file.
 
-Sentry does not install a JSON library nor an HTTP client by itself. Sentry will default to the [built-in `JSON`](https://hexdocs.pm/elixir/JSON.html) for JSON and [Hackney] for HTTP requests, but can be configured to use other ones. To use the default ones, do:
+Sentry does not install a JSON library nor an HTTP client by itself. Sentry will default to the [built-in `JSON`](https://hexdocs.pm/elixir/JSON.html) for JSON and [Finch] for HTTP requests, but can be configured to use other ones. To use the default ones, do:
 
 ```elixir
 defp deps do
   [
     # ...
 
     {:sentry, "~> 10.8"},
-    {:hackney, "~> 1.20"}
+    {:jason, "~> 1.4"},
+    {:finch, "~> 0.19"}
```

Comment:
Nit, but let's suggest a version requirement that wouldn't technically allow breaking versions (since this is before 1.0):

```suggestion
    {:finch, "~> 0.19.0"}
```